pkgs/applications/misc/parsec/generic.nix

```diff
@@ -23,34 +23,58 @@
 , vulkan-loader
 , gnome
 , zenity ? gnome.zenity
+, darwin
```

Comment:
is `darwin` used here?

Reply:
This expression isn't using the `darwin` argument so it can be safely removed